App.js

```diff
@@ -3,15 +3,52 @@ import {StyleSheet, Text, View} from 'react-native';
 
 import Header from './header';
 import Footer from './footer';
+import { PRIORITIES } from './constants';
+
+// type Props = {};
+export default class App extends Component { // <Props> {
+  constructor(props) {
+    super(props);
+    this.state = {
+      text: "",
+      priority: PRIORITIES.MEDIUM,
+      items: []
+    }
+    this.handleAddItem = this.handleAddItem.bind(this);
+  }
+
+  handleAddItem() {
+    if (!this.state.text) return; // ignore empty strings
+    const newItems = [
+      ...this.state.items,
+      {
+        key: Date.now(),
+        text: this.state.text,
+        priority: this.state.priority,
+        complete: false
```

Comment:
Mogao bi propery da se zove `isCompleted` ili `completed` :D Bolje nego da bude glagol

Reply:
ovo sam ispravio na "done", ali je to na drugoj grani :D
Ispraviću i ovdje :)